torch/ao/quantization/qconfig_mapping.py

```diff
@@ -106,16 +106,6 @@ def _get_default_qconfig_mapping(is_qat: bool, backend: str, version: int) -> QC
             fixed_qparams_observer_to_qconfig[observer] = fixed_qparams_qconfig
         qconfig_mapping.set_object_type(fixed_qparams_op, fixed_qparams_qconfig)
 
```

Comment:
could you please add a comment here summarizing what you said in the summary? it will be helpful for future reference when we have the new api ready

Reply:
Ok It's been added.